renderer-macos/lib/bindings/BriskEffectView.re

```diff
@@ -0,0 +1,90 @@
+
+module Material = {
+  type t;
+  external titlebar: unit => t =
+    "ml_getNSVisualEffectMaterialTitlebar_bc"
+    "ml_getNSVisualEffectMaterialTitlebar";
+  let titlebar = titlebar();
```

Comment:
I'm not sure if we should call those function at the top level, thoughts?

Reply:
How about good old polymorphs instead of directly calling this?

Reply:
I think they are unnecessary here for the props. Polymorphic variants are great for styles because of well, polymorphism but a bit unnecessary for props.

Reply:
Unless this should be a style!

Reply:
Done!